### PR TITLE
feat(sampling): Add rules panel new design - [TET-164]

### PR DESCRIPTION
**Pull request by priscilawebdev**
**What this PR does:**

- Add a table that is very similar to the dynamic sampling one but with cleanups
- Add the new "Active" column to the sampling rules panel
- Add New Test

**Preview:**

https://user-images.githubusercontent.com/29228205/176703034-311f9194-553b-45ff-bbd8-96320fa5e7a5.mov

### CODE REVIEW

File: static/app/components/switchButton.tsx [hunk 1]
```diff
@@ -43,6 +44,7 @@ const Switch = ({
     isActive={isActive}
     size={size}
     data-test-id="switch"
+    {...props}
```
Comment (priscilawebdev) [inline static/app/components/switchButton.tsx:47]: it was needed for the `aria-label`

File: static/app/types/sampling.tsx [hunk 2]
```diff
@@ -167,9 +167,14 @@ export type SamplingRule = {
...
-   * A rule can be disabled if it doesn't contain a condition (Else case)
+   * Indicates if the rule is enabled for server-side sampling
    */
-  disabled?: boolean;
+  active?: boolean;
+  /**
+   * A rule without a condition (Else case) always have to be 'pinned'
+   * to the bottom of the list and cannot be sorted.
+   */
+  bottomPinned?: boolean;
```
Comment (priscilawebdev) [inline static/app/types/sampling.tsx:177]: Change this because it was weird having `active` and `disabled`

**Review by matejminar — APPROVED** [review]
1. Are we hiding the active column on mobile on purpose?

2. Looks like uniformRateModal needs to be updated with the new docs url

3. There's a weird behaviour with actions:

https://user-images.githubusercontent.com/9060071/176882339-6249384a-0c50-4b2d-abfe-fbd1b017e3e6.mp4

 Not a blocker for wip.

**Comment by priscilawebdev** [comment]
> 1. Are we hiding the active column on mobile on purpose?

  I see the active column as an important action to have on mobile, but it takes some space, making conditions and rate shrink, so I was wondering if we should move it to the actions button. In any case, we will still have to think of a good UI for mobile cc @Jesse-Box 

> 2. Looks like uniformRateModal needs to be updated with the new docs url

I have just updated the code..thanks! 

> There's a weird behaviour with actions:

I noticed the same but decided to take a look into it another time and create a separate PR. Not sure if we have the same behaviour in production atm

**Pull request merged**